native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -0,0 +1,179 @@
+      if (node.attribs.href) {
+        console.debug(`Found href: ${decodeURI(node.attribs.href)}`)
+        const newResource = cacheDictionary[decodeURI(node.attribs.href)]
+        if (newResource) {
+          console.debug(`Replaced ${node.attribs.href} with ${newResource}`)
+          node.attribs = { ...(node.attribs || {}), href: newResource }
+          return node
+        }
+      } else if (node.attribs.src) {
+        console.debug(`Found src: ${decodeURI(node.attribs.src)}`)
+        const newResource = cacheDictionary[decodeURI(node.attribs.src)]
+        if (newResource) {
+          console.debug(`Replaced ${node.attribs.src} with ${newResource}`)
+          node.attribs = { ...(node.attribs || {}), src: newResource }
+          return node
+        }
+      }
+    }
+  }
+
+  // see https://github.com/archriss/react-native-render-html/issues/286
+  renderBulletLists: RendererFunction = (htmlAttribs, children, convertedCSSStyles, passProps) => {
+    const { language } = this.props
+    const style = _constructStyles({
+      tagName: 'ul',
+      htmlAttribs,
+      passProps,
+      styleSet: 'VIEW'
+    })
+    const { allowFontScaling, rawChildren, nodeIndex, key, baseFontStyle, listsPrefixesRenderers } = passProps
+    const baseFontSize = baseFontStyle.fontSize
+    children = children && children.map((child, index) => {
+      const rawChild = rawChildren[index]
+      let prefix = false
+      const rendererArgs = [
+        htmlAttribs,
+        children,
+        convertedCSSStyles,
+        {
+          ...passProps,
+          index
+        }
+      ]
+      const rtlMargin = 10
+      const bulletSizeRelativeToFont = 2.8
+      if (rawChild) {
+        if (rawChild.parentTag === 'ul' && rawChild.tagName === 'li') {
+          prefix = listsPrefixesRenderers && listsPrefixesRenderers.ul ? listsPrefixesRenderers.ul(...rendererArgs) : (
+            <View style={{
+              marginRight: 10,
+              width: baseFontSize / bulletSizeRelativeToFont,
+              height: baseFontSize / bulletSizeRelativeToFont,
+              marginTop: baseFontSize / 2,
+              borderRadius: baseFontSize / bulletSizeRelativeToFont,
+              backgroundColor: 'black',
+              marginLeft: RTL_LANGUAGES.includes(language) ? rtlMargin : 0
+            }} />
+          )
+        } else if (rawChild.parentTag === 'ol' && rawChild.tagName === 'li') {
+          prefix = listsPrefixesRenderers && listsPrefixesRenderers.ol ? listsPrefixesRenderers.ol(...rendererArgs) : (
+            <Text allowFontScaling={allowFontScaling} style={{
+              marginRight: 5,
+              fontSize: baseFontSize,
+              marginLeft: RTL_LANGUAGES.includes(language) ? rtlMargin : 0
+}}>{index + 1})</Text>
+          )
+        }
+      }
+      return (
+        RTL_LANGUAGES.includes(language) ? (
+          <View key={`list-${nodeIndex}-${index}-${key}`} style={{ flexDirection: 'row', marginBottom: 10 }}>
```

Comment:
Maybe you could make the margin 10 a global variable as you are using it here as well.

Reply:
removed it, as this margin was related to the text distance in lists and in my opinion we do not actually need it.